src/main/java/de/qabel/desktop/ui/remotefs/RemoteFSController.java

```diff
@@ -410,8 +411,12 @@ private Pane buttonFromImage(TreeItem<BoxObject> item, HBox bar, ImageView butto
 
     private ReadableBoxNavigation createSetup() throws QblStorageException {
         volume = boxVolumeFactory.getVolume(clientConfiguration.getAccount(), clientConfiguration.getSelectedIdentity());
-        nav = volume.navigate();
-
+        try {
+            nav = volume.navigate();
+        } catch (QblStorageNotFound e) {
+            volume.createIndex("meh");
```

Comment:
hm.. what else? think we should remove this parameter from the core but it's oos. I should have integrated the new core in a seperate PR .. but now it's too late :'(